vm/systemSmartContracts/delegationManager.go

```diff
@@ -313,7 +313,7 @@ func (d *delegationManager) getAllContractAddresses(args *vmcommon.ContractCallI
 		return vmcommon.UserError
 	}
 
-	for _, address := range contractList.Addresses {
+	for _, address := range contractList.Addresses[1:] {
```

Comment:
You should add a check that len(contractList.Addresses)>0, otherwise this will panic.

Reply:
at initialization it will be at least one. I can add for extra security.